enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,18 +172,34 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+
+        if requested_emails := request.data.get('learner_emails'):
+            request_data = {}
+            if act_by_date := request.data.get('act_by_date'):
```

Comment:
do we need a conditional here? Could we just set the `request_data` to the result of `request.data.get('act_by_date')`